### PR TITLE
chore(infra-apps): bump Argo CD chart to 1.17.*

### PR DESCRIPTION
**Pull request by hairmare**
# Description
Update to an Argo CD chart version that contains Arrgo CD 1.8.4.

Below are the changes to the chart, please see the [release notes](https://github.com/argoproj/argo-cd/releases) for details on the content of the Argo CD update.

## Features:
* enable support for lifecycle hooks on ArgoCD Server
* conditionally create the rbac configmap
* update to 1.8.4
* support for readOnlyRootFilesystem in dex
* Mount ephemeral emptyDir on repoServer for /tmp and /app/config/gpg/keys
* add self-heal-timeout as a template variable
* adds configEnabled flag to allow parent chart to own CM
* upgrade redis-ha to v4.10.4
* add spec for the status field of AppProject CRD
* add possibility for change NodePort if this is type of Service
* add annotations to the dex service account
* create roles instead of rolebindings when singleNamespace is true

## Fixes:
* replace hard coded .selector.matchLabels (`-application-controller` suffix is now templates via `{{ .Values.controller.name }}`)

# Issues

# Checklist

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run -a`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: charts/infra-apps/Chart.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ name: infra-apps
 description: Argo CD app-of-apps config for infrastructure components
 type: application
 # version and appVersion are in sync in this chart!
-version: 0.9.2
-appVersion: 0.9.2
+version: 0.9.3
+appVersion: 0.9.3
```
Comment (tongpu) [inline charts/infra-apps/Chart.yaml:7]: What's our policy regarding version bumps in the chart? Up to now I thought we're bumping the minor version of the chart when we increase the minor version of any subcharts?

Comment (hairmare) [reply charts/infra-apps/Chart.yaml]: This should have absolutely been a bump at the minor level!

**Review by tongpu — APPROVED** [review]
LGTM

File: charts/infra-apps/Chart.yaml (hunk 1 above)
Comment (eyenx) [reply charts/infra-apps/Chart.yaml]: Do we want to write that down somewhere in our documentation?

**Pull request merged**